articles/hibernate-one-to-one-mapping/index.md

```diff
@@ -0,0 +1,573 @@
+
+## Table of contents
+---
+- Setting up the development environment.
+- Creating the database and configuring hibernate configuration file.
+- Creating the entities.
+- Creating and executing the main app.
+
+## Prerequisites
+---
+Basic java knowledge working with [object oriented programming](https://www.w3schools.com/java/java_oop.asp#:~:text=Java%20%2D%20What%20is%20OOP%3F,contain%20both%20data%20and%20methods.&text=OOP%20provides%20a%20clear%20structure%20for%20the%20programs), how to implement [composition](https://www.geeksforgeeks.org/composition-in-java/#:~:text=The%20composition%20is%20a%20design,that%20refers%20to%20other%20objects.) between entities, [SQL](https://www.w3schools.com/sql/sql_intro.asp) commands, [MySQL](https://dev.mysql.com/doc/mysql-installation-excerpt/5.7/en/mysql-installer-setup.html) shell/bash, and Intellij but feel free to use any environment preferred.
+
+## Step 1: Setting up the development environment
+---
+Install MySQL server using the following link.
+
+- [MySQL installation guide](https://dev.mysql.com/doc/mysql-installation-excerpt/5.7/en/mysql-installer-setup.html)
+
+After successfully installing MySQL and setting up database credentials, use the following command to log in to MySQL.
+
+```MySQL
+mysql -u username -p
+```
+A prompt will ask for your password and on entering access will be granted showing the information below.
+
+![MySQL login](mysql-login.png)
+
+ Maven is a build tool for managing java projects. Maven is based on the project object model (POM) which is an XML file containing configuration details and information about the project. The dependencies can be obtained from [maven central repository](https://mvnrepository.com/) and added to the POM file where maven will be responsible for source code generation, code compilation, and JAR files packaging of the compiled code in a local repository where they will be accessed when a new maven project is created. The required dependencies will be downloaded automatically by maven and added to the classpath of your project once they are added to the POM file.
+
+In IntelliJ click file, new, project, and on the window that opens select maven, then click next and give your project a name. This will create a new maven project which has the following structure. 
```

Comment:
this is insufficient, intellij requires more advanced setup and installation. Tell the reader to use Eclipse as it has hibernate archetypes that the reader may find helpful in creating/starting hibernate projects

Reply:
> this is insufficient, intellij requires more advanced setup and installation. Tell the reader to use Eclipse as it has hibernate archetypes that the reader may find helpful in creating/starting hibernate projects

@Manley12 I have fixed the requested changes. Thank you!